src/components/menu/menu.test.js

```diff
@@ -279,7 +279,7 @@ context("Testing Menu component", () => {
       submenu().eq(positionOfElement("first"), div).trigger("mouseover");
       innerMenu(positionOfElement("second"), span).click({ multiple: true });
       for (let i = 0; i < 3; i++) {
-        cy.focused().tab();
+        cy.focused().trigger("keydown", keyCode("Tab"));
```

Comment:
not sure I see why this needed changing - `.tab()` is more readable and I assume does the same thing. But I'm no cypress expert - if the `.tab()` stopped working and this rewrite works then *shrug*

Reply:
This was one of the flaky cypress tests that I mentioned a few days ago, for some reason it didn't like iterating and calling tab on the focused element but passes with this refactor. I'm happy to let @DlgSHi look into it as I can't work out why it's being inconsistent